code/modules/clothing/outfits/factions/syndicate.dm

```diff
@@ -54,10 +54,10 @@
 	box = /obj/item/storage/box/survival/syndie
 
 /datum/outfit/job/syndicate/assistant/gorlex
```

Comment:
We should probably change the paths on these outfit datums to syndicate/%JOB%/hardliner for clarity's sake. The plain /gorlex datums should be reserved for "legacy" outfits meant to represent actual ICW-era personnel (like for corpses). probably just with the old tactical turtlenecks and such for now, til we get some Old Gorlex recolors at some point. Just don't think it would be fitting for ICW gorlex to be decked out in white.

Reply:
I'll get around to that later as this is mostly a stopgap. Legacy Gorlex outfits will exist together with Ramzi.